cdap-docs/developers-manual/source/building-blocks/applications.rst

```diff
@@ -10,16 +10,16 @@ Applications
 
 .. highlight:: java
 
-An **Application** is a collection of application virtualizations that read from—and write
-to—the data virtualization layer in CDAP. 
+An **Application** is a collection of building blocks that read—and write—through the data
```

Comment:
I thought we would say: ...that read and write data through the data abstraction layer of CDAP